specs-go/config.go

```diff
@@ -16,8 +16,9 @@ type Spec struct {
 
 // Device is a "Device" a container runtime can add to a container
 type Device struct {
-	Name           string         `json:"name"`
-	ContainerEdits ContainerEdits `json:"containerEdits"`
+	Name           string            `json:"name"`
+	Annotations    map[string]string `json:"annotations,omitempty"` // Annotations add meta information per device
```

Comment:
Probably good to mention that this metadata is not affecting container metadata.

Reply:
Is there a difference between what is here and:
```
// Annotations add meta information per device. Note these are CDI-specific and do not affect container metadata.       
Annotations map[string]string `json:"annotations,omitempty"` 
```

Reply:
My comment was about make it explicit that those annotations are not going to be resulted in editing OCI container spec. So, @elezar suggested edit is good one in my opinion.

Reply:
Good point.Will update.